tasks/qcow2_image.yml

```diff
@@ -188,8 +188,8 @@
             auth: "{{ ovirt_auth }}"
           register: resized_disk
           until: resized_disk.disk.provisioned_size != ovirt_disk.disk.provisioned_size
-          retries: 30
-          delay: 2
+          retries: "{{ (disk_resize_timeout / 3) | int }}"
+          delay: 3
```

Comment:
@machacekondra I believe "delay" should be a variable as well.
disk_resize_timeout: 60
disk_resize_delay: 3
          retries: "{{ (disk_resize_timeout / disk_resize_delay) | int }}"
	  delay: "{{ disk_resize_delay }}"

Reply:
why?